test/test_gnoi.py

```diff
@@ -21,6 +21,17 @@ def test_gnoi_reboot(self):
         assert ret == 0, 'Fail to read counter'
         assert new_cnt == old_cnt+1, 'DBUS API is not invoked'
 
+    def test_gnoi_reboot_halt(self):
+        ret, old_cnt = gnmi_dump('DBUS config reload')
```

Comment:
Do you mind explaining what these dumps (DBUS config reload, DBUS API not invoked") are for? They do not seem related to the test case.